CHANGELOG.rst

```diff
@@ -25,6 +25,7 @@ Changes
 * Add average metric aggregation for elastic search
 * Bugfix to query ordering in Elasticsearch TermsGroupBy
 * Added all parameters for StringColumnStyle
+* Add derivative metric aggregation for Elasticsearch
```

Comment:
Can I have you rebase this and move this comment up to the "next version" section?